conf/messages.cy

```diff
@@ -69,6 +69,7 @@ subpage.utr = Cyfeirnod Unigryw y Trethdalwr (UTR)
 subpage.sidebar.stoptrading = Rhoi’r gorau i fasnachu
 subpage.sidebar.dormant = Troi eich cwmni''n segur
 subpage.sidebar.close = Cau eich cwmni
+subpage.sidebar.dormant.close = Caewch eich cwmni neu gwnewch yn segur
```

Comment:
subpage.sidebar.dormant_close is consistent with our agreed standard

Reply:
oh never mind i understand now
 :D